### DISCUSSION TITLE
turtle in codewars

### DISCUSSION THREAD

**Discussion by Anuragdullur** [codewars/codewars.com#2617] [Q&A] [answered]
How to import turtle lib in Codewars??
I am trying to solve a KATA using turtle function it is throwing me an error "import of inspect halted; None in sys.modules"
can someone help on this...

**Comment by Blind4Basics** [comment, marked as answer]
you cannot. This module isn't installed in the current setup, server side. You can find all the available modules [here](https://docs.codewars.com/languages/python#packages)

**Reply by Anuragdullur** [reply]
Thanks :)
